cmd/vcfFilter/vcfFilter.go

```diff
@@ -46,6 +69,15 @@ func vcfFilter(infile string, outfile string, c criteria, groupFile string, pars
 			v = vcf.ParseFormat(v, header)
 		}
 
+		if c.biAllelicOnly {
+			if sitesSeen[v.Chr][v.Pos] < 1 {
```

Comment:
In my sisyphean struggle to convince people that `switch` > `else if` I will point out that in addition to its obvious stylistic superiority, here it would also have the added benefit of only 2 lookups instead of 4.

```
switch sitesSeen[v.Chr][v.Pos] {
     case 0:  // Panic
     case 1:  // continue
     default: // remove site
}
```